Gemfile.prod

```diff
@@ -0,0 +1,20 @@
+ruby '~> 2.3.0'
+
+eval_gemfile 'Gemfile.base'
+
+source 'https://gems.contribsys.com/' do
+  gem 'sidekiq-pro', '~> 3.4', require: %w(sidekiq-pro sidekiq/pro/web)
+end
+
+# one of the license terms does not permit modifications and presents unclear risk to redhat
+gem 'airbrake', '~> 4.3.3'
+
+# one of the license terms does not permit modifications and presents unclear risk to redhat
+# NewRelic RPM
+# docs says it should be loaded latest as possible
+group :production, :preview, :development do
+  gem 'newrelic-redis'
+  gem 'newrelic_rpm', '~>3.5'
+  gem 'rpm_contrib'
+end
+
```

Comment:
We should probably add
```ruby
set :bundle_gemfile, 'Gemfile.prod'
```
to https://github.com/3scale/deploy/blob/master/system/config/deploy.rb after that.